consensus/genesis.go

```diff
@@ -142,7 +143,7 @@ func MakeGenesisFunc(opts ...GenOption) GenesisInitFunc {
 				return nil, err
 			}
 		}
-		if err := SetupDefaultActors(ctx, st, storageMap); err != nil {
+		if err := SetupDefaultActors(ctx, st, storageMap, proofs.TestMode); err != nil {
```

Comment:
BLOCKING: Are you sure that you want to default this to `proofs.TestMode`? It seems like this should be configured via a `GenOption`.

Reply:
This function is only run in test scenarios, so it makes sense to default to `TestMode`